linux-anvil-cuda/Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN ldconfig -v 2>/dev/null | grep -v ^$'\t' | cut -f1 -d":" >> /etc/ld.so.conf.
 ADD https://loripsum.net/api /opt/docker/etc/gibberish
 
 # Resolves a nasty NOKEY warning that appears when using yum.
-RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-${CENTOS_VER}
+RUN find /etc/pki/rpm-gpg/ -type f | xargs -I {} rpm --import "{}"
```

Comment:
Needed as this file wasn't present. Also there were a couple others in there that should be imported